typescript/scripts/README.adoc

```diff
@@ -0,0 +1,38 @@
+:toc: macro
+
+= tBTC v2 Refund script
+
+This script explains what to do in case you need to refund your BTC.
+
+toc::[]
+
+== Refund script
+
+=== Prerequisites
+
+Please make sure you have the following prerequisites installed on your machine:
+
+- https://nodejs.org[Node.js] >14.18.0
+- https://yarnpkg.com[Yarn] >1.22.19
+
+tBTCv2 system prerequisite is that you can refund your BTC only after `refundLocktime` expires, which is currently set to 9 month.
```

Comment:
Let's make sure we don't exceed 80 characters per line.

Reply:
done